File: QE-Selenium-Java/qe-selenium-java/qe-selenium-java/src/main/java/demos/Activity5.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity5 {
public static void main(String[] args) {
    

    WebDriver driver = new FirefoxDriver();
    driver.get("https://training-support.net/webelements/dynamic-controls");
    System.out.println(driver.getTitle());
    System.out.println(driver.findElement(By.xpath("//input[@id='checkbox']")).isDisplayed());
    driver.findElement(By.xpath("//button[contains(text(),'Toggle Checkbox')]")).click();;
    System.out.println(driver.findElement(By.xpath("//input[@id='checkbox']")).isDisplayed());
    driver.quit();

}
    }

File: Qe-Selenium-Python/qe-selenium-python/Activity5.py
from selenium import webdriver
from selenium.webdriver.common.by import By
with webdriver.Firefox() as driver:
     driver.get("https://training-support.net/webelements/dynamic-controls")
     print(driver.title)
     print(driver.find_element(By.XPATH,"//input[@id='checkbox']").is_displayed())
     driver.find_element(By.XPATH,"//button[contains(text(),'Toggle Checkbox')]").click()
     print(driver.find_element(By.XPATH,"//input[@id='textInput']").is_displayed())
     driver.quit()